src/controllers/comment.ts

```diff
@@ -0,0 +1,36 @@
+import { Request, Response } from 'express';
+import {
+	create,
+	postComments,
+	deleteComment,
+	commentReplies,
+	createReply,
+} from '../lib/comment';
+
+const createComment = async (req: Request, res: Response) => {
+	const response = await create(req, res);
+};
+
+const getPostComments = async (req: Request, res: Response) => {
+	const response = await postComments(req, res);
```

Comment:
Please don't pass req and res to service layer

Reply:
Done